### ISSUE TITLE
[Perspectives - Keyboard (and others)] WCAG 2.0 Related Link

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-showcase-examples#223]
In the Learn More section. The WCAG 2.0 link to the related success criteria in the QuickRef does not work. It just goes to the QuickRef with no filters applied.

I redid the filters and copied the share link and tried it in a new browser tab and it still did not work. I believe this is an issue with the QuickRef share this view link.

This was the case for these WCAG2.0 links on all Video pages.

**Comment by shawna-slh** [comment]
@yatil - quickref bug?

**Comment by yatil** [comment]
Works for me – although I noticed some delay on the W3C server that slowed things down. Tags are applied here. I tested in Safari and Firefox. If you see something else on other browsers, please let me know in the [w3c/wai-wcag-quickref issues](http://github.com/w3c/wai-wcag-quickref/issues/).

**Comment by bakkenb** [comment]
I was using Chrome on Windows and it does not work. Just tried FF and it worked fine. Thanks for pointing that out @yatil. Need to either fix for Chrome or report as known bug in the QuickRef

**Comment by yatil** [comment]
Works for me with Chrome on OS X, too. Maybe it is holding onto an old JavaScript, Brent, try a shift+reload on the quickref, please.

**Comment by yatil** [comment]
^^ @bbakken

**Comment by bakkenb** [comment]
@yatil  - That fixed it. It is now working on my machine in Chrome. Thank you.

@nitedog - This issue can be closed.

**Comment by nitedog** [comment]
Thank you @yatil
